pkg/webhook/resources/setting/validator.go

```diff
@@ -1306,6 +1307,29 @@ func (v *settingValidator) checkStorageNetworkRangeValid(config *storagenetworkc
 	if !network.IP.Equal(ip) {
 		return fmt.Errorf("Range should be subnet CIDR %v", network)
 	}
+
+	var ipAddrListExclude []string
+	ipAddrListInclude := webhookutil.GetUsableIPAddressesInclude(network)
+	//Minimum ips required is 2.
+	//single node with default LH pods(backing-image,instance-manager)
+	if len(ipAddrListInclude) < 2 {
+		return fmt.Errorf("no allocatable IP addresses, check IP range for include and exclude in storage network config")
```

Comment:
..only %v IPs are avaiable and they are less then the minimal %v IPs...

Reply:
This check is basically to avoid conditions where user can provide /32 or /31 range ip address where /32 provides no allocatable addresses and /31 provides only 1 ip address.
(network and broadcast addresses are not included in allocatable address range by whereabouts)

Reply:
We may define the min as e.g. 2 or 16, and the log message shows clearly this error.

Min value 2 only makes sense for test; a roll-upgrade can also break single-node. We may define it as 8 or 16.